test/test_rules.py

```diff
@@ -4,92 +4,116 @@
 
-    def test_if_three_in_diagonal_top_right_to_bottom_left_return_true(self):
-        rules = Rules()
-        current_board = ["1", "2", "X", "4", "X", "6", "X", "8", "9"]
-        winner = rules.is_winner_diagonal(current_board)
-        self.assertEqual(True, winner)
+        self.assertEqual(True, self.rules.is_winner_diagonal(top_left_btm_right))
+        self.assertEqual(True, self.rules.is_winner_diagonal(top_right_btm_left))
 
     def test_if_not_three_in_diagonal_row_return_false(self):
-        rules = Rules()
+
         current_board = ["1", "X", "X", "4", "X", "6", "7", "8", "9"]
-        winner = rules.is_winner_diagonal(current_board)
+        winner = self.rules.is_winner_diagonal(current_board)
         self.assertEqual(False, winner)
 
-    def test_if_winner_diagonal_is_winner_return_true(self):
-        rules = Rules()
-        current_board = ["1", "2", "X", "4", "X", "6", "X", "8", "9"]
-        is_winner_true = rules.is_winner(current_board)
-        self.assertEqual(True, is_winner_true)
-
-    def test_if_winner_horizontal_is_winner_return_true(self):
-        rules = Rules()
-        current_board = ["1", "2", "3", "4", "5", "6", "X", "X", "X"]
-        is_winner_true = rules.is_winner(current_board)
-        self.assertEqual(True, is_winner_true)
-
-    def test_if_winner_vertical_is_winner_returns_true(self):
-        rules = Rules()
-        current_board = ["1", "X", "3", "4", "X", "6", "7", "X", "9"]
-        is_winner_true = rules.is_winner(current_board)
-        self.assertEqual(True, is_winner_true)
+    def test_is_winner_returns_true_if_winner_in_any_direction(self):
+        horizontal_winner_board = ["1", "2", "3", "4", "5", "6", "X", "X", "X"]
+        vertical_winner_board = ["1", "X", "3", "4", "X", "6", "7", "X", "9"]
+        diagonal_winner_board = ["1", "2", "X", "4", "X", "6", "X", "8", "9"]
+        self.assertEqual(True, self.rules.is_winner(horizontal_winner_board))
+        self.assertEqual(True, self.rules.is_winner(vertical_winner_board))
+        self.assertEqual(True, self.rules.is_winner(diagonal_winner_board))
 
     def test_if_not_winner_is_winner_returns_false(self):
-        rules = Rules()
         current_board = ["1", "X", "3", "4", "X", "X", "7", "8", "9"]
-        is_winner_true = rules.is_winner(current_board)
+        is_winner_true = self.rules.is_winner(current_board)
         self.assertEqual(False, is_winner_true)
+
+    def test_top_row_returns_top_row_of_board(self):
+        top_row_board = self.rules.top_row(self.starter_board)
+        self.assertEqual(["1", "2", "3"], top_row_board)
+
+    def test_middle_row_returns_middle_row_of_board(self):
+        middle_row_board = self.rules.middle_row(self.starter_board)
+        self.assertEqual(["4", "5", "6"], middle_row_board)
+
+    def test_bottom_row_returns_bottom_row_of_board(self):
+        bottom_row_board = self.rules.bottom_row(self.starter_board)
+        self.assertEqual(["7", "8", "9"], bottom_row_board)
+
```

Comment:
Here is an example of combining the tests into one concept.  Just like a function, a test should do one thing and do it well.  What you have is not incorrect.  This feedback is to point out that the definition of one-thing can sometimes include multiple assertions.

```suggestion
    def test_can_identify_rows_of_the_board(self):
        top_row_board = self.rules.top_row(self.starter_board)
        middle_row_board = self.rules.middle_row(self.starter_board)
        bottom_row_board = self.rules.bottom_row(self.starter_board)
        
        self.assertEqual(["1", "2", "3"], top_row_board)
        self.assertEqual(["4", "5", "6"], middle_row_board)
        self.assertEqual(["7", "8", "9"], bottom_row_board)
```